ocs_sample_library_preview/Securable.py

```diff
@@ -0,0 +1,217 @@
+        """
+        Patch an item's access control list
+        :param item_id: The item identifier
+        :param patch: A JSON Patch document describing the patch operations
+        """
+        if item_id is None:
+            raise TypeError
+        if patch is None:
+            raise TypeError
+
+        response = self.__base_client.request(
+            'patch', self.__item_acl_path.format(
+                tenant_id=self.__tenant,
+                namespace_id=namespace_id,
+                item_id=item_id),
+            data=patch.to_string())
+        self.__base_client.checkResponse(
+            response, f'Failed to patch access control, {item_id} in collection {self.__collection}.')
+
+    def getOwner(self, namespace_id: str, item_id: str) -> Owner:
+        """
+        Get an item's owner
+        :param item_id: The item identifier
+        """
+        if item_id is None:
+            raise TypeError
+
+        response = self.__base_client.request(
+            'get', self.__item_owner_path.format(
+                tenant_id=self.__tenant,
+                namespace_id=namespace_id,
+                item_id=item_id))
+        self.__base_client.checkResponse(
+            response, f'Failed to get owner, {item_id} in collection {self.__collection}.')
+
+        result = Owner.fromJson(response.json())
+        return result
+
+    def updateOwner(self, namespace_id: str, item_id: str, owner: Owner):
+        """
+        Update an item's access control list
+        :param item_id: The item identifier
+        :param owner: The owner
+        """
+        if item_id is None:
+            raise TypeError
+        if owner is None:
+            raise TypeError
+
+        response = self.__base_client.request(
+            'put', self.__item_owner_path.format(
+                tenant_id=self.__tenant,
+                namespace_id=namespace_id,
+                item_id=item_id),
+            data=owner.toJson())
+        self.__base_client.checkResponse(
+            response, f'Failed to update access control, {item_id} in collection {self.__collection}.')
+
+    def getAccessRights(self, namespace_id: str, item_id: str) -> list[CommonAccessRightsEnum]:
+        """
+        Get an item's access rights
+        :param item_id: The item identifier
+        """
+        if item_id is None:
+            raise TypeError
+
+        response = self.__base_client.request(
+            'get', self.__item_access_rights_path.format(
+                tenant_id=self.__tenant,
+                namespace_id=namespace_id,
+                item_id=item_id))
+        self.__base_client.checkResponse(
+            response, f'Failed to get access rights, {item_id} in collection {self.__collection}.')
+        
+        content = response.json()
+        result = []
+        if len(content) > 0:
+            for i in content:
+              result.append(CommonAccessRightsEnum[i])
```

Comment:
Would this be a good time to add the new 'Share' access right to this enum?
https://docs.osisoft.com/bundle/ocs/page/developer-guide/identity-dev/dev-guide-role-based-access-control.html

Reply:
I added the Share access right since it is in the docs and it should not be a breaking change.